src/app/components/UserAssets/Vesting/VestingWithdrawForm.tsx

```diff
@@ -0,0 +1,147 @@
+};
+
+export const VestingWithdrawForm: React.FC<VestingWithdrawFormProps> = ({
+  vesting,
+  onClose,
+}) => {
+  const { t } = useTranslation();
+  const account = useAccount();
+  const { value, loading } = useGetUnlockedVesting(
+    vesting.staking,
+    vesting.vestingContract,
+  );
+  const [address, setAddress] = useState(account);
+
+  const { send, ...tx } = useSendToContractAddressTx(
+    vesting.vestingContract,
+    VestingAbi as AbiItem[],
+    'withdrawTokens',
+  );
+  const handleSubmit = useCallback(() => {
+    if (!tx.loading) {
+      send(
+        [address.toLowerCase()],
+        { from: account },
+        { type: TxType.SOV_WITHDRAW_VESTING },
+      );
+    }
+  }, [account, address, send, tx]);
+
+  const [scheduleOpen, setScheduleOpen] = useState(false);
+  const openSchedule = useCallback(event => {
+    event.preventDefault();
+    setScheduleOpen(true);
+  }, []);
+  const closeSchedule = useCallback(() => setScheduleOpen(false), []);
+
+  return (
+    <>
+      <div className={styles.container}>
+        <div className={styles.wrapper}>
+          <h2 className={styles.title}>
+            {t(translations.vestingDialog.title)}
+          </h2>
+          <p>{t(translations.vestingDialog.subtitle)}</p>
+          <FieldGroup label={t(translations.vestingDialog.amount)}>
+            <DummyField>
+              <div className="tw-w-full tw-flex tw-justify-between tw-items-center tw-relative">
+                <div className="tw-w-full tw-flex-grow tw-text-center">
+                  {loading
+                    ? t(translations.vestingDialog.calculating)
+                    : weiTo4(value)}
+                </div>
+                <div
+                  className={classNames(
+                    'tw-flex-shrink tw-flex-grow-0 tw-absolute',
+                    styles.right,
+                  )}
+                >
+                  <AssetSymbolRenderer asset={vesting.asset} />
+                </div>
+              </div>
+            </DummyField>
+            <div className="tw-text-right tw-mt-2">
+              <span
+                className="tw-link tw-cursor-pointer tw-text-xs"
+                onClick={openSchedule}
+              >
+                {t(translations.vestingDialog.schedule)}
+              </span>
+            </div>
+          </FieldGroup>
+          <div className="tw-mx-auto tw-text-center">
+            <img
+              src={arrowDown}
+              alt="Arrow Down"
+              className={styles.arrowDown}
+            />
+          </div>
+          <FieldGroup
+            label={t(translations.vestingDialog.receiver, {
+              asset: vesting.asset,
+            })}
+          >
+            <InputField
+              onChange={event => setAddress(event.target.value)}
+              value={address}
+            />
+          </FieldGroup>
+
+          <div className={styles.txFee}>
+            {t(translations.common.fee, { amount: '0.000014' })}
```

Comment:
Do you want to hardcode the fee?

Reply:
Yes, due to the bug on rsk node this call will almost always estimate far less gas than required sk value is hardcoded to the maximum function would use

Reply:
ok, thanks for the explanation